### ISSUE TITLE
Aircraft not communicating in voice

### ISSUE THREAD

**Issue by Hawaiicoder** [openscope/openscope#1582]
Browser: Chrome

Airport: LAX

Approximate play time before issue: ~5m

#### Issue Description
_please include any console errors and/or screenshots_ 
Aircraft were responding in text but voice was skipped.

#### Steps to reproduce
- unknown

**Comment by OlaRune** [comment]
I also have this problem in Chrome at different airports: Seen it on both Oslo and Hamburg. When it happens I can turn off the speech option and turn it back on again and it starts working. After a while it stops working again and the option has to be toggled again.

**Comment by JonasCir** [comment]
For me, speech does not work at all (both Firefox + Chrome on Linux).

**Comment by OlaRune** [comment]
There is a (probably) related issue here. #1489

**Comment by JonasCir** [comment]
I solved my problem by installing `espeak-ng` and enabling it as output module for `speech-dispatcher` under `/etc/speech-dispatcher/speechd.conf`. Maybe this should be added to a trouble shoot section? :)
Not certain about speech stopping after some time though...

**Comment by JonasCir** [comment]
@OlaRune 
B/c of your questions in Slack, see my last comment :)

**Comment by OlaRune** [comment]
> @OlaRune 
> 
> B/c of your questions in Slack, see my last comment :)

Thanks! I installed espeak-ng and tried to add it as a module. However, I get a log error when it tries to run. I'll look into it further when I have more time. Thanks for the hint.

**Comment by JonasCir** [comment]
`AddModule "espeak-ng"                "sd_espeak-ng" "espeak-ng.conf"`
Thats what I have

**Comment by OlaRune** [comment]
> `AddModule "espeak-ng"                "sd_espeak-ng" "espeak-ng.conf"`
> 
> Thats what I have
> 
> I have not experimented which other voices are working :D I was just happy to hear anything :D

That's what I did. My problem right now is that when I'm doing `spd-say testing` from the command line it says that it can't find a Unix Domain Socket to connect to and that it can't autospawn (something). I guess it's a permission problem or something.

**Comment by JonasCir** [comment]
`sudo systemctl start speech-dispatcherd`  or `enable` instead of `start` if you want it persisted

**Comment by OlaRune** [comment]
> ah: `sudo systemctl start speech-dispatcherd`  or `enable` instead of `start` if you want it persisted
> 
> 

Yep, I've tried that but the service crashes without return code. ☺️

**Comment by JonasCir** [comment]
oh :deer:
